lib/robots/dor_repo/etd_submit/check_marc.rb

```diff
@@ -16,7 +16,8 @@ def initialize(opts = {})
 
         #========= This method check symphony and looks for MARC record based on druid ==========#
         def perform(druid)
-          etd = Etd.find(druid)
+          object_client = Dor::Services::Client.object(druid)
+          etd = object_client.find
```

Comment:
It looks to me like this value gets passed to `Etd::IdentityMetadataGenerator.generate`, which requires an ActiveFedora Etd object.

Reply:
so this should be left as `Etd.find()`?  or switch to `Dor::Etd.find` or `Dor::Item.find` or whatever?